config/routes/project.rb

```diff
@@ -15,7 +15,7 @@
     scope '-' do
       get :activity
       get :edit
-      post :transfer
+      resources :transfer, only: %i[create new]
```

Comment:
As per comment above, let's undo this change and just use the `transfer` endpoint instead.

Reply:
Reverted in 2703d62